build.sh

```diff
@@ -58,42 +97,44 @@ EOF
 XCODE_XCCONFIG_FILE="${PWD}/Carthage/64bit.xcconfig" carthage bootstrap --cache-builds --platform macOS
 
 echo ""
-echo "$(tput setaf 2)Building and archiving$(tput sgr 0)"
+echo "Building and archiving"
 xcodebuild archive -project eduVPN.xcodeproj -scheme $TARGET -archivePath $FILENAME.xcarchive DEVELOPMENT_TEAM=$TEAMID
 
 echo ""
-echo "$(tput setaf 2)Exporting$(tput sgr 0)"
+echo "Exporting"
 /usr/libexec/PlistBuddy -c "Set :teamID \"$TEAMID\"" ExportOptions.plist
 xcodebuild -exportArchive -archivePath $FILENAME.xcarchive -exportPath $FILENAME -exportOptionsPlist ExportOptions.plist
 
 echo ""
-echo "$(tput setaf 2)Re-signing up and down scripts$(tput sgr 0)"
+echo "Re-signing up and down scripts"
 DOWN=$(find $FILENAME -name "*.down.*.sh" -print)
 codesign -f -s "$SIGNINGIDENTITY" "$DOWN"
 UP=$(find $FILENAME -name "*.up.*.sh" -print)
 codesign -f -s "$SIGNINGIDENTITY" "$UP"
 
 echo ""
-read -p "Create disk image (requires DropDMG license) (y/n)?" choice
-case "$choice" in
-  y|Y ) ;;
-  n|N ) exit 0;;
-  * ) echo "Invalid response."; exit 0;;
-esac
 
-echo ""
-echo "$(tput setaf 2)Creating a disk image$(tput sgr 0)"
-# The configuration eduVPN can be used for all products
-echo "Using: dropdmg --config-name \"eduVPN\" --signing-identity=\"$SIGNINGIDENTITY\" \"$FILENAME/$PRODUCT\""
-dropdmg --config-name "eduVPN" --signing-identity="$SIGNINGIDENTITY" "$FILENAME/$PRODUCT"
+INSTALLERFILENAME="eduVPN-Installer$(date +"%Y-%m-%d-%H:%M:%S.")dmg"
 
-echo ""
-echo "$(tput setaf 2)Creating app cast XML$(tput sgr 0)"
-DISTRIBUTIONPATH="../eduvpn-macos-distrib"
-# Assumptions are being made about the location of this script
-# Also, this often fails due to extended attribute
-echo "Using: $DISTRIBUTIONPATH/generate_appcast $DISTRIBUTIONPATH/dsa_priv.pem $DISTRIBUTIONPATH/updates/"
-$DISTRIBUTIONPATH/generate_appcast $DISTRIBUTIONPATH/dsa_priv.pem $DISTRIBUTIONPATH/updates/
+
+create-dmg \
+--volname "eduVPN" \
+--volicon "icon.icns" \
```

Comment:
Don't use eduVPN logo for the volume.

Reply:
What is the problem with putting eduVPN icon here?

Reply:
The icon in icon.icns is for an app, not for a volume. Also, it's the wrong icon if you build for Let's Connect target.

Reply:
Done, since script can now put two different icon